### ISSUE TITLE
[compile error] Can't compile pcl with RelWithDebInfo on windows (missing /bigobj flag)

### ISSUE THREAD

**Issue by Faroito** [PointCloudLibrary/pcl#5811]
**Describe the error**

When compiling pcl 1.13.0 on Windows with cmake option `CMAKE_BUILD_TYPE=RelWithDebInfo`, I have this error:

```
pcl\segmentation\src\organized_multi_plane_segmentation.cpp : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj
```

**To Reproduce**

- clone pcl 1.13.0
- build it with the `CMAKE_BUILD_TYPE=RelWithDebInfo` option on windows

**Your Environment (please complete the following information):**

 - OS: Windows 11
 - Compiler: MSVC 14.29.30133
 - PCL Version 1.13.0
 - PCL Type: compile with source

If PCL was compiled from source or failure in compiling PCL itself:
 - CMAKE_BUILD_TYPE=RelWithDebInfo

**Possible Solution**

I have added manually the cxx flag `/bigobj` and it compiles.

**Comment by mvieth** [comment]
@Faroito Are we talking about PCL 1.13.0 or PCL 1.13.1?

**Comment by mvieth** [comment]
@Faroito Please try 1.13.1 (or the current master branch) then, the problem should be fixed there.

**Comment by Faroito** [comment]
Yes, it's fixed for 1.13.1. Thanks!
